apps/web/pages/_document.tsx

```diff
@@ -23,3 +43,72 @@ export default function Document() {
     </Html>
   );
 }
+
+Document.getInitialProps = async (
+  ctx: DocumentContext,
+): Promise<CustomDocumentProps & DocumentProps> => {
+  const initialProps = await ctx.defaultGetInitialProps(ctx);
+  const { pathname } = ctx;
+
+  // Define default metadata
```

Comment:
Thanks for doing this refactor! I think this will feel more maintainable if we store data in data structures rather than code, meaning we could define something like:

```
const ogDataForPath = {
   '/about': { title: 'Base | About', description: 'From the beginning', etc. },
   '/bootcamp': {...},
}
```

and then this whole function can just be

```
  return {
    ...initialProps,
    metadata: ogDataForPath[pathname] ?? defaultOgData,
  };
  ```
  
  I bet U can prompt an LLM to do this refactor